WMF Framework/EventLoggingStandardEventProviding+EventLoggingEventValuesProviding+EventLoggingCategory+EventLoggingLabel.swift

```diff
@@ -1,6 +1,6 @@
-public protocol EventLoggingEventValuesProviding {
+@objc public protocol EventLoggingEventValuesProviding {
```

Comment:
Changed it to be available in objc to let `WMFExploreViewController` conform and provide the values. In `WMFArticleViewController` I'm checking if `articlePreviewingActionsDelegate` conforms to `EventLoggingEventValuesProviding` and log the provided values if it does. `WMFArticleViewController` handles peek & pop for a couple different controllers and I needed to log different values based on the origin of the peeked article.